### PR TITLE
Self Checkout and Terms of Service Pages Added

### PR DESCRIPTION
**Pull request by sahilakos**

### CODE REVIEW

File: assets/js/checkout.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+            // Required to remove overflow set by the modal
+            // Same as: https://lodash.com/docs/4.17.10#debounce
+            setTimeout(function() { $body.removeAttr('style'); }, 0);
+          }
+        }
+      });
+    }, 1250);
+  }
+
+  _updateCheckout();
```
Comment (brikis98) [inline assets/js/checkout.js]: Code looks reasonable... But I'd be terrified of changing it. There's lots more logic here in our own page than I realized!

Can we add automated tests around this? At the minimum, unit tests for the various functions, but ideally, integration tests that use Chargebee in a checkout flow too. Otherwise, the very next update is very likely to break this flow... And that's obviously a problem.

File: pages/checkout/_subscription.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+              </h5>
+              <div class="row">
+                <div class="col-xs-12 col-sm-6 col-sm-offset-3 col-md-12 col-md-offset-0">
+                  <ul class="check-list small text-white margin-bottom-none">
+                    <li><span class="help-text light" title="250,000+ lines of reusable, battle-tested infrastructure code." data-toggle="tooltip">Infrastructure as Code Library</span></li>
+                    <li><span class="help-text light" title="A collection of training video courses that teach you DevOps." data-toggle="tooltip">DevOps Training Library</span></li>
+                    <li><span class="help-text light" title="Mission Control for your infrastructure. Currently in private beta." data-toggle="tooltip">Gruntwork Houston</span></li>
+                    <li><span class="help-text light" title="Get support via our public community forum. 2 Business Day SLA." data-toggle="tooltip">Community Forum</span></li>
+                    <li class="check-list-disabled" id="subscription-addon-1">Dedicated Support</li>
+                    <li class="check-list-disabled" id="subscription-addon-2">Reference Architecture</li>
```
Comment (brikis98) [inline pages/checkout/_subscription.html]: Support and Ref Arch should have the dotted underline for consistency

File: pages/signup-complete/index.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
+---
+layout: default
+title: Signup Complete!
+permalink: /signup-complete/
+slug: signup-complete
+sitemap: false
+noindex: true
```
Comment (brikis98) [inline pages/signup-complete/index.html]: 👍

File: pages/terms/index.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+          </div>
+        </div>
+        <div class="col-xs-12 visible-md-block visible-lg-block">
+            <div class="row terms-head">
+                <h5 class="h2 col-md-5">Plain English</h5>
+                <h5 class="h2 col-md-7">Legalese</h5>
+            </div>
+        </div>
+        <div class="col-xs-12 tos-table">
+          {% for category in site.data.terms-of-service %}
```
Comment (brikis98) [inline pages/terms/index.html]: Thx for putting the terms in a data file. Would be painful to edit a gigantic HTML file.

File: pages/checkout/_subscription.html (hunk 2 above)
Comment (josh-padnick) [reply pages/checkout/_subscription.html]: +1. @sahilakos Could you add this?

Comment (sahilakos) [reply pages/checkout/_subscription.html]: Will do.
